javascript/javascript3/week2/fetchingAndWaiting.js

```diff
@@ -0,0 +1,31 @@
+
+//promise way 
+// @ts-ignore
+const promise = new Promise(resolve => {
+    setTimeout(() => {
+        resolve();
+    }, 3000);
+})
```

Comment:
The variable `promise` could have a more meaningful name or it could be a function like
```js
function waite(seconds) {
  return new Promise(resolve => {
    setTimeout(() => {
      resolve();
    }, seconds * 1000);
  });
}
```
Then you can use it like this
```js
wait(8).then(...)
```